Dockerfile.develop

```diff
@@ -15,9 +15,9 @@
 ###############################################################################
 # Create the develop, test, and build environment
 ###############################################################################
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
```

Comment:
Should we go bigger here and jump to UBI 9.1.0. I had tried that a week ago and it seemed to work fine.

Reply:
@njhill suggested to stick with UBI `8.7` so this is good :-)